Documentation/ceph-object-bucket-claim.md

```diff
@@ -102,18 +102,79 @@ metadata:
   labels:
     aws-s3/object [1]
 provisioner: rook-ceph.ceph.rook.io/bucket [2]
-parameters: [3]
+parameters:
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
+  region: us-west-1 [3]
   bucketName: ceph-bucket [4]
 reclaimPolicy: Delete [5]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
-1. **all** `parameter` required.
+1. `region`(optional) defines the region in which the bucket should be created. For RGW, this maps to [zonegroup](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) of the server. If user does not define `region`, Rook will fill this value accordingly. Please check example at the end of the doc.
```

Comment:
Shall we mention the default here? Otherwise, I'm not sure what it means for Rook to "fill this value".
```suggestion
1. `region`(optional) defines the region in which the bucket should be created. For RGW, this maps to [zonegroup](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) of the server. If user does not define `region`, the default is `us-east-1`. Please check example at the end of the doc.
```

Reply:
@travisn : I don't what I missed earlier if the user does not define the region, Rook will fill it from `Zonegroup` from `ceph object store context` and don't use `us-east-1` at all(at least that was my first intention of PR).

`us-east-1` is not the default value from RGW(at least not in RGW code base). It is a value that always works mysteriously. Maybe because a lot of s3 clients use this as default for `region`, but again not defined as default by RGW

Reply:
Ok, does that mean rook can just take care of this internally? Why do we need to expose the `region` setting at all?

Reply:
@travisn : The `region` defined in `storage class` of libbucket provisioner. Not sure whether Rook can hide it from user, maybe we can ignore the value. But this was available earlier release not sure whether it right to remove the variable. AFAIR I have sent https://github.com/rook/rook/pull/8766 as per request from the user, he wants to set the custom region for his use case.